apps/api/src/report/report.service.ts

```diff
@@ -26,7 +27,7 @@ export class ReportService {
     private readonly ienapplicantStatusRepository: Repository<IENApplicantStatus>,
   ) {}
 
-  captureFromTo(from: string, to: string) {
+  captureFromTo(from: string, to?: string) {
     this.reportUtilService._isValidDateValue(from);
     this.reportUtilService._isValidDateValue(to);
     if (!from) {
```

Comment:
wonder if we could just do something like `to ? this.reportUtilService._isValidDateValue(to) : to = dayjs().format('YYYY-MM-DD')`, if `to` is undefined, I don't see the need to call the isValidDate function

Reply:
I want to vote for readability because I am quite excellent at writing codes that are hard to read.